controller/errror.controller.js

```diff
@@ -0,0 +1,7 @@
+exports.handlePSQLErrors = (err, req, res, next) => {
```

Comment:
Rename errror to error - will save you trouble later if you spell it correctly now :) Also - error.controller isn't really a controller - split this file out into a new folder called errors and put your error handling in that folder as a new index.js file.